File: DayTwo/JavaScript/LevelThree/TaskSeven.js
const prompt = require("prompt-sync")();

let result = 0;
        
for(let count = 1; count <= 5; count++){
        
number = Number(prompt("Enter a number: "));
            
    result += number;
}
console.log("The sum of the 5 numbers = " + result);

File: DayTwo/Java/LevelThree/TaskSeven.java
import java.util.Scanner;

public class TaskSeven{
    public static void main(String... args){
    
    Scanner input = new Scanner(System.in);

    int result = 0;
        
        for(int count = 1; count <= 5; count++){
        
            System.out.print("Enter a number: ");
            int number = input.nextInt();
            
        result += number;
        }
        System.out.println("The sum of the 5 numbers = " + result);
    }
}
